catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -99,6 +99,13 @@
             default="false"
             required="false"/>
 
+        <AD id="cacheDisabled"
+            name="Disable Cache"
+            description="Disables use of cache."
```

Comment:
This was copied from the search ui configuration but I think we could update it to be clear what's going on.